browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1679,6 +1680,51 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 		return false;
 	},
+	
+	_linkButtonControl: function(parentContainer, data, builder) {
```

Comment:
best to put new controls in the new file and already in TS, but we can move that later